shared/vcjs/verifyCredential.ts

```diff
@@ -70,17 +71,35 @@ export async function verifyCredential(
   }
 }
 
-function handleResponse(result: any) {
+function handleResponse(
+  result: any,
+  verifiableCredential: VerifiableCredential | Credential,
+) {
   var errorMessage = VerificationErrorType.NO_ERROR;
   var isVerifiedFlag = true;
 
   if (!result?.verified) {
-    if (result['results'][0].error.name == 'jsonld.InvalidUrl') {
+    let errorCodeName = result['results'][0].error.name;
+    if (errorCodeName == 'jsonld.InvalidUrl') {
       errorMessage = VerificationErrorType.NETWORK_ERROR;
+      isVerifiedFlag = false;
+    } else if (errorCodeName == VerificationErrorType.RANGE_ERROR) {
+      errorMessage = VerificationErrorType.RANGE_ERROR;
+      const vcIdentifier = verifiableCredential.credentialSubject.UIN
+        ? verifiableCredential.credentialSubject.UIN
+        : verifiableCredential.credentialSubject.VID;
+      sendErrorEvent(
+        getErrorEventData(
+          TelemetryConstants.FlowType.vcVerification,
+          TelemetryConstants.ErrorId.vcVerificationFailed,
+          TelemetryConstants.ErrorMessage.vcVerificationFailed + vcIdentifier,
+        ),
+      );
+      isVerifiedFlag = true;
     } else {
       errorMessage = VerificationErrorType.TECHNICAL_ERROR;
```

Comment:
we can get rid of else by setting this as default error.
 ```
if (!result?.verified) {
    errorMessage = VerificationErrorType.TECHNICAL_ERROR;
    isVerifiedFlag = false;

    if (errorCodeName == 'jsonld.InvalidUrl') { 
    // update error code 
    } else if (errorCodeName == VerificationErrorType.RANGE_ERROR) {
     // logic for range error 
    }
 }
```

Reply:
refactored it.